pkg/dbutil/common.go

```diff
@@ -617,3 +624,18 @@ func ReplacePlaceholder(str string, args []string) string {
 	newStr := strings.Replace(str, "?", "'%s'", -1)
 	return fmt.Sprintf(newStr, utils.StringsToInterfaces(args)...)
 }
+
+// ExecSQLWithRetry executes sql with retry
+func ExecSQLWithRetry(ctx context.Context, db *sql.DB, sql string, args ...interface{}) (err error) {
+	ctx1, cancel := context.WithTimeout(ctx, DefaultTimeout)
+	defer cancel()
+
+	for i := 0; i < DefaultRetryTime; i++ {
+		_, err = db.ExecContext(ctx1, sql, args...)
+		if err == nil {
```

Comment:
some errors will never success (like table already exists), doing retry have no meaning. Should we need to add something like `isRetryableError`?